src/core/credentials/transport/tls/grpc_tls_certificate_distributor.h

```diff
@@ -171,7 +178,8 @@ struct grpc_tls_certificate_distributor
   // root certs, while pem_root_certs still contains the valid old data.
   struct CertificateInfo {
     // The contents of the root certificates.
-    std::string pem_root_certs;
+    std::variant<std::string, std::shared_ptr<grpc_core::SpiffeBundleMap>>
+        roots;
     // The contents of the identity key-certificate pairs.
     grpc_core::PemKeyCertPairList pem_key_cert_pairs;
     // The root cert reloading error propagated by the caller.
```

Comment:
Not directly related to this PR, but we should probably change the representation here to use `absl::StatusOr<>` instead of having a separate `grpc_error_handle` field for each of the root cert and identity cert errors.  (This code was originally written before `grpc_error_handle` was an alias for `absl::Status`.)

Feel free to add a TODO if you don't want to try to tackle this now.